src/content/pages/index.de.mdx

```diff
@@ -125,15 +125,15 @@ import FaqSection from "../../components/widgets/faq/Wrapper.astro";
     {
       title: "Closed beta",
       version: "3.β",
-      variant: "inprogress",
+      variant: "check",
       body: `Vereinzelte Bugs. Ab dieser Version werden aber die Daten erhalten und die Datenbank nicht mehr zurückgesetzt.
        Kann für echte aber nicht zeitkritische Lagerplanung verwendet werden.
         Die beschränkte Anzahl an Betatestenden müssen mit Wartungsunterbrüchen rechnen.`,
     },
     {
       title: "MVP: Lager",
       version: "3.0",
-      variant: "todo",
+      variant: "inprogress",
```

Comment:
Are we really at this step? Aren't we technically already done with 3.0 now, and the course features 3.1 are in progress after the go-live? Or otherwise, when do we make that transition?